lib/ui/discover/HeroListItem.dart

```diff
@@ -71,4 +72,11 @@ class HeroListItem {
       child: NetworkImageFromFuture(articleData.imageUrl, fit: BoxFit.fitWidth),
     );
   }
+
+  Future _onMenuPressed(BuildContext context) async {
```

Comment:
don't test things like this...... just create a MockViewModel, don't mess with the widgets and add temp code, you can already pass in the function due to the view model! 

Revert this file.

Davids debug demo widget will make this easy